src/index.js

```diff
@@ -0,0 +1,71 @@
+import React from "react";
+import ReactDOM from "react-dom";
+import { Menu } from "./components/Menu";
+import { OrderSummary } from "./components/Order";
```

Comment:
If you add an `index.js` to the root of the `components` folder, you can export both `Menu` and `OrderSummary`. Then you can combine the imports into `import { Menu, OrderSummary } from './components`